python/delta_sharing/converter.py

```diff
@@ -21,6 +21,41 @@
 import pandas as pd
 
 
+def _get_dummy_column(type_string: str):
+    if type_string == "boolean":
+        return pd.Series([False])
+    elif type_string == "byte":
+        return pd.Series([0], dtype="int8")
+    elif type_string == "short":
+        return pd.Series([0], dtype="int16")
+    elif type_string == "integer":
+        return pd.Series([0], dtype="int32")
+    elif type_string == "long":
+        return pd.Series([0], dtype="int64")
+    elif type_string == "float":
+        return pd.Series([0], dtype="float32")
+    elif type_string == "double":
+        return pd.Series([0], dtype="float64")
+    elif isinstance(type_string, str) and type_string.startswith("decimal"):
```

Comment:
nit: Do we need to check `isinstance(type_string, str)`?

Reply:
Ryan used this syntax for `to_converter` so I just ran with it. Slacked Ryan about this just now.

Reply:
Checked with Ryan, no need to check for `isinstance`